docs/core/card-view.component.md

```diff
@@ -74,6 +74,10 @@ Defining properties from Typescript:
         key: 'mental-stability',
         default: 0.0
     }),
+    new CardViewVariablesItemModel({
```

Comment:
I don't think the name selection is correct here. In AMA we would like to use this new feature of CardView component to display and manage process variables, however this variables concept are only exists in AMA.
In reality, this component is supposed to give a solution for a more general key-value pair concept.
I suggest to rename this to CardViewKeyValuePairsItemModel for example.